lib/Fhp/Options/FinTsOptions.php

```diff
@@ -45,6 +45,14 @@ class FinTsOptions
     /** @var int */
     public $timeoutResponse = 30;
 
+    /**
+	 * The Kundensystem-Id as returned by the bank and persisted by the application code
+	 * Prevents having to re-authenticate every time on login
+	 * Use DialogInitialization::getKundensystemId() on the return-object of FinTs::login(), to get the new one
+	 * @var string
```

Comment:
Make it clear by the type here and in the explanation above that this is optional. And give it a `null` default value.